src/classes/CW_BatchUpdateAudienceGroupMemberTest.cls

```diff
@@ -0,0 +1,109 @@
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.